demos/src/demo.js

```diff
@@ -47,10 +45,10 @@ function loadDemo(showtype) {
 					m = rule.match(/^([^\:]+)\:\s*\(([^\)]+)\)/);
 					if (m) {
 						if (m[2].match(/_deprecated\s*:/)) return true;
-						var op = '<div data-o-grid-colspan="12 M6 L3" class="demo-sample"><div class="demo-swatch';
-						var tips = [];
+						let op = '<div data-o-grid-colspan="12 M6 L3" class="demo-sample"><div class="demo-swatch';
+						const tips = [];
 						roles.forEach(function(role) {
-							var rolematch = (new RegExp(role+"\\s*:\\s*([\\w-]+),")).exec(m[2]+',');
+							const rolematch = (new RegExp(role+'\\s*:\\s*([\\w-]+),')).exec(m[2]+',');
```

Comment:
does it not throw an error with the `+` concatenator? I got a lot of errors in obt saying that it needed to be prefixed and suffixed with a space